src/scripts/exportCSV.js

```diff
@@ -0,0 +1,93 @@
+export function displayExportButton() {
+  const weekCard = document.getElementById("week-card-content");
+
+  //get export button div & clear if re-generated
+  const parentContainer = document.getElementById("export-button-div");
+  parentContainer.innerHTML = "";
+
+  // Create the button
+  const exportButton = document.createElement("button");
+  exportButton.className = "export-button";
+  exportButton.id = "export-button";
+  exportButton.type = "button";
+  exportButton.textContent = "Download CSV 🗒";
+  parentContainer.appendChild(exportButton);
+
+  weekCard.appendChild(parentContainer);
+
+  //convert HTML table to CSV on click of the button
+  exportButton.addEventListener("click", function () {
+    tableToCSV();
+  });
+}
+
+export function tableToCSV() {
+  // Variable to store the final csv data
+  let csv_data = [];
+
+  //add custom header to CSV
+  csv_data.push(
+    "Group, (Intern 1) Name, (Intern 1) Department, (Intern 1) Location, (Intern 2) Name, (Intern 2) Department, (Intern 2) Location",
+  );
+
+  // Get each row data by getting table and starting after header
+  let table = document.getElementById("interns-week-table");
+  let rows = table.getElementsByTagName("tr");
```

Comment:
not being re-assigned so should be`const` also i think we can remove the intermediate variable
```suggestion
  const rows = document.getElementById("interns-week-table").getElementsByTagName("tr");
```